interpreter/CMakeLists.txt

```diff
@@ -404,15 +404,15 @@ if(builtin_llvm)
       MESSAGE(FATAL_ERROR "Cannot extract LLVM version number (${versionpart}) from ${llvmconfigfile}")
     endif()
   endforeach()
-  set(LLVM_VERSION "${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}.${LLVM_VERSION_PATCH}" PARENT_SCOPE)
+  set(LLVM_VERSION "${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}.${LLVM_VERSION_PATCH}" CACHE INTERNAL "")
```

Comment:
Did you really need to change that? It yes, could you confirm that it works in both builtin_llvm ON and OFF?

Reply:
@vgvassilev we need these changes because otherwise LLVM_VERSION variable is used in core/metacling for creating a directory where clang includes will be installed and it will not be visible within PARENT_SCOPE in this case and directory can't be created.